### ISSUE TITLE
Comments are not justified

### ISSUE THREAD

**Issue by lbarman** [CourseAdvisor/courseadvisor#116]
Comments are left-aligned; long comments look slightly bad this way (e.g. http://courseadvisor.ch/fr/course/it-security-engineering-tp-991).  I think justifying the text would look better.

**Comment by hmil** [comment]
Good point. You can submit a PR together with a before/after screenshot to highlight the enhancement.
Take a look at assets/style/review.less ;)

**Comment by lbarman** [comment]
Perhaps we should poll existing users for their opinion before making such a big change.
